.github/ISSUE_TEMPLATE/task.md

```diff
@@ -1,14 +1,95 @@
 ---
 name: New Task
 label: task
-assignees: ''
+assignees: '@.rai (CoderBlee)'
 about: 'Create a new task to be tracked'
 ---
 
-# Task Title
+# Task Title: BlackPythonDevs Hackathon & Learning Fest
+## Date & Time:
+- **Date:** [Choose a Date]
+- **Time:** [Choose a Time] (e.g., 10:00 AM – 4:00 PM)
+## Venue:
+- Location: [Choose a Venue or Virtual Platform]
+- Address: [Physical Address if in-person, or platform details if virtual]
 
-Due Date: %date
+## Tasks - Do these in Order
 
-Description:
+### Objective:
+To bring together Python, hackerthors and absolute beginners enthusiasts to collaborate on open-source projects, learn new skills, and network with like-minded individuals.
```

Comment:
```suggestion
To bring Python enthusiasts, hackathon participants, and absolute beginners in collaborative open-source projects, fostering skill development and creating opportunities for networking with like-minded individuals.
```

Reply:
Rewording to bring your point home.

Reply:
Thank you soo much for looking into it. I was discouraged and honestly did not know what to do anymore. I also learned not to contribute in bulk